### PR TITLE
FIX #448 Make the probe-dictionary search results header sticky

### LINKED ISSUE #448: Make the probe-dictionary search results header sticky
In the [probe-dictionary search results](https://telemetry.mozilla.org/probe-dictionary/), it would be great if the table header is always in view, no matter where you scroll to.

Chris mentioned that [`position: sticky`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) might help here.

### PR DESCRIPTION
**Pull request by gargshruti30**
FIX #448 Make the probe-dictionary search results header sticky.
This PR solves this issue .
I have made the changes in a new branch 'sticky'.
@chutten Please review this PR.

### CODE REVIEW

File: probe-dictionary/explorer.css [hunk 1]
```diff
@@ -131,3 +131,8 @@ body {
 .nav-item {
     font-weight: bold;
 }
+th{
```
Comment (chutten) [inline probe-dictionary/explorer.css]: The style in this file is to have a full line of whitespace between rulesets, and a space between the selector and `{`

File: probe-dictionary/explorer.css [hunk 2]
```diff
@@ -131,3 +131,8 @@ body {
...
+    position: sticky;
+    top: -1px;
```
Comment (chutten) [inline probe-dictionary/explorer.css]: Why `-1`? Would `0` not make more sense, or is it convention?

**Review by chutten — CHANGES_REQUESTED** [review]

File: probe-dictionary/explorer.css (hunk 1 above)
Comment (gargshruti30) [reply probe-dictionary/explorer.css]: @chutten I have corrected it.

File: probe-dictionary/explorer.css (hunk 2 above)
Comment (gargshruti30) [reply probe-dictionary/explorer.css]: @chutten It is because when I put it 0 there was some visible space left.

**Comment by gargshruti30** [comment]
@chutten Can you please now review this PR.

**Review by chutten — APPROVED** [review]
Excellent, thank you for your contribution.

**Pull request merged**